varnishgather

```diff
@@ -428,6 +436,40 @@ done
 
 shift $((OPTIND - 1))
 
+INSTANCE_NAMES=$(get_varnishd_instance)
+if [ -n "$NAME" ]; then
+    INSTANCE_NAME="${NAME#-n }"
+   
+    if ! echo "$INSTANCE_NAMES" | grep -qx "$INSTANCE_NAME"; then
```

Comment:
this is sketchy because an unnamed instance can still be pinged using `-n`, and you can use an absolute `-n` to ping a relatively named instance. As long as the workdir can be fine it work.

However, I don't foresee users doing twisted things like that, this is fine